packages/wallet/ui/public/bridge.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <script src="%PUBLIC_URL%/lockdown.umd.js"></script>
+    <title>Agoric Wallet Bridge</title>
+</head>
+
+<body style="overflow-x: hidden; overflow-y: scroll">
+    <noscript>Without JavaScript enabled, this page cannot function.</noscript>
```

Comment:
that statement is true invariantly. minor but let's tell the user something about the state.
```suggestion
    <noscript>ERROR: JavaScript not available.</noscript>
```

Reply:
Fair enough. Better to be explicit and tell them how to fix it?

I have a hard time blaming the user when the page requires JavaScript. I was trying to concisely say something like "Please excuse the requirement for you to put yourself at risk by enabling JavaScript in your browser. This page uses JavaScript for more than progressive enhancement. Providing similar functionality without JavaScript is not practical."

Reply:
Just used the same message as index.html. Do people still use the web without javascript? At the very least they're trying to use a dapp so I don't think we need to be apologetic.

Reply:
> ... so I don't think we need to be apologetic.

No, we don't need to, but...

> Do people still use the web without javascript?

Yes. In fact, I bet you do it regularly, with `curl` and the like.